backend/models/vueformGenerator.ts

```diff
@@ -52,15 +52,25 @@ function mapSchemaTypeToVueformElement(
     vueformElement['rules'].push('max:' + schemaType.max)
   }
 
-  if (schemaType.description !== undefined) {
-    vueformElement['description'] = translate('description.' + labelStr, language)
+  if (schemaType.description) {
+    vueformElement['description'] = translate(schemaType.description, language)
+  } else if (labelStr) {
+    let description = translate('description.' + labelStr, language) as { de: string; en: string }
+    if (description.de) {
```

Comment:
Die if Bedingung habe ich hinzugefügt, um zu verhindern, dass für alle Schematypen eine Description angezeigt wird, die nur aus dem Translationkey besteht, wenn in den Locales nichts eingetragen ist.